### PR TITLE
Appdata related paches

### PR DESCRIPTION
**Pull request by sabriunal**
### appdata: Add categories and keywords support

These tags may improve Software Center accessibility.

More information: https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-categories

### appdata: Use appstreamcli for appdata validation

appstream-util is obsoleted by appstreamcli.

### data: Update runtime version

appstreamcli available on GNOME 45 runtime.

### CODE REVIEW

**Comment by sabriunal** [comment]
Michael Catanzaro recommended the use of appstreamcli in [this comment](https://gitlab.gnome.org/World/eartag/-/merge_requests/95#note_1874726).

**Comment by sabriunal** [comment]
There is an ['appstream-util'](https://github.com/maoschanz/drawing/blob/master/debian/control#L5) entry in debian/control that I have no idea how to handle.

File: data/com.github.maoschanz.drawing.appdata.xml.in [hunk 1]
```diff
@@ -33,6 +33,17 @@
     <p>Supported file types include PNG, JPEG and BMP.</p>
   </description>
 
+  <categories>
+      <category>Graphics</category>
```
Comment (sabriunal) [inline data/com.github.maoschanz.drawing.appdata.xml.in:37]: space?

**Comment by sabriunal** [comment]
appstream package contain appstreamcli binary. 

- https://packages.ubuntu.com/focal/appstream
- https://packages.ubuntu.com/focal/amd64/appstream/filelist

**Review by maoschanz — APPROVED** [review]

**Pull request merged**